xdsl/dialects/arith.py

```diff
@@ -551,6 +456,9 @@ def get(operand1: Union[Operation, SSAValue],
             result_types=[IntegerType.from_width(1)],
             attributes={"predicate": IntegerAttr.from_int_and_width(arg, 64)})
 
+    def verify_(self) -> None:
+        pass
```

Comment:
By default, the `verify_` function should do nothing, so we don't have to add it here

Reply:
right